playwright/utils.ts

```diff
@@ -2,6 +2,6 @@ const localBaseUrl = 'http://localhost:7777/';
 
 const referenceBaseUrl = 'https://guardian.github.io/commercial-templates/';
 
-const templatePreviewWidths = ['360', '740', '980', '1300', '100%'];
+const templatePreviewWidths = ['740', '980', '1300', '100%'];
```

Comment:
Why did we remove the 360 width from the playwright snapshots?

Reply:
Not a permanent change, sorry! Just playing around with the tests atm as they're a little flaky